templates/federated_reporting/import.sh

```diff
@@ -24,7 +24,11 @@ fi
 
 # TODO: we should do some validation of the files here
 mkdir -p "$CFE_FR_SUPERHUB_IMPORT_DIR"
-mv "$CFE_FR_SUPERHUB_DROP_DIR/"*".sql.$CFE_FR_COMPRESSOR_EXT" "$CFE_FR_SUPERHUB_IMPORT_DIR" || true
+mv "$CFE_FR_SUPERHUB_DROP_DIR/"*".sql.$CFE_FR_COMPRESSOR_EXT" "$CFE_FR_SUPERHUB_IMPORT_DIR" ||
+  {
+    log "No files in drop dir."
+    exit 0
```

Comment:
Maybe let's not `exit` here, just in case there are some "leftovers" in the `IMPORT_DIR` to give them a chance to be picked up?

Reply:
Sure thing. I was also wondering if logging here is really good. Maybe too much noise for a "normal" sort of case. So feel free to reject and move on. :)

Reply:
>  Maybe too much noise for a "normal" sort of case. So feel free to reject and move on. :)

I think logging here is okay. Definitely at this phase, maybe we will do some changes for the LTS version of FR.